package/yapsy/__init__.py

```diff
@@ -53,7 +53,7 @@
 
 """
 
-__version__="1.10.423"
+__version__="1.10.423.py_9"
```

Comment:
I did this to test the integration with my system but I forgot to revert the change. Sorry!